packages/swapper/src/swappers/thorchain/utils/makeSwapMemo.ts

```diff
@@ -0,0 +1,43 @@
+import { adapters } from '@shapeshiftoss/caip'
+
+import { SwapError, SwapErrorTypes } from '../../../api'
+
+// BTC (and likely other utxo coins) can only support up to 80 character memos
+const MAX_LENGTH = 80
+
+/**
+ * @returns thorchain memo shortened to a max of 80 characters as described:
```

Comment:
better link to show memo message limits
https://dev.thorchain.org/thorchain-dev/memos#mechanism-for-transaction-intent